src/_P008_RFID.ino

```diff
@@ -159,7 +168,7 @@ boolean Plugin_008(uint8_t function, struct EventStruct *event, String& string)
             // read a tag
             Plugin_008_keyBuffer = Plugin_008_keyBuffer >> 1; // Strip leading and trailing parity bits from the keyBuffer
 
-            keyMask = (static_cast<uint64_t>(0x1) << (PCONFIG(0) - 2));    // Shift in 1 just past the number of remaining bits
+            keyMask = (0x1ull << (PCONFIG(0) - 2));           // Shift in 1 just past the number of remaining bits
             keyMask--;                                        // Decrement by 1 to get 0xFFFFFFFFFFFF...
```

Comment:
This is strange code.
Are we sure it is correct?
For 26 bit code, you end up with 24x a 1. (2^24 -1 or 0xFF FFFF)
Why 24 bits masked and not 26?

Reply:
Ah... now I see, it is stripping the leading and trailing parity bit...

I need coffee...

Reply:
Hmm... those are not used...

Reply:
It starts by removing the parity bit at the 'right-side' (shift at line 169), then uses a mask to chop off the parity bit at the 'left-side'. Similar code was there before, but with fixed masks for 26 and 34 bit frames (24 and 32 bit payloads), now that the frame-size is variable, I have to build the mask dynamically in some way, and this seemed the simplest way, shift a bit to the left then decrement by 1 to get all 1s 😉

Reply:
> Hmm... those are not used...

Adding a parity check I'd prefer to do in a new PR, with a tester (or a Wiegand RFID scanner) available, and then hope to catch communication errors... 😛

Reply:
An easier way to mask it is this:
```c++
keyMask = ~(static_cast<uint64_t>(0x1) << (PCONFIG(0) - 1));
```
This explicitly sets the leading partity bit to 0 when performing a logical AND.
But leave it as it is now, as that's tested and proven to work :)